_posts/2016-09-23-custom-flows-parsing-xml-part-2.md

```diff
@@ -51,7 +51,7 @@ new GraphStageLogic(shape) with InHandler with OutHandler {
        val coalesced = buffer.toString()
        isBuffering = false
        buffer.clear()
-       emit(out, Characters(coalesced), () => emit(out, other))
+       emitMultiple(out, Iterable(Characters(coalesced), other))
```

Comment:
Docs target Scala devs, while the blog is mixed. I would prefer List even though `:: Nil` is more idiomatic for the same reason :)